dask_expr/expr.py

```diff
@@ -810,6 +810,26 @@ def _task(self, index: int):
         else:
             return (self.operation,) + tuple(args)
 
+    def _simplify_up(self, parent):
+        if isinstance(
+            self,
+            (
+                Replace,
+                Isin,
+                Clip,
+                ToTimestamp,
+                IsNa,
+                Round,
+                Abs,
+                Apply,
+                Map,
+            ),
+        ) and isinstance(parent, Projection):
+            if list(self.columns) == list(parent.columns):
+                return self
+
+            return type(self)(self.frame[parent.operand("columns")], *self.operands[1:])
```

Comment:
Dusting off my "OO Programmer" hat (a rarely used hat) I'd say that this breaks the idea of inheritance, where the superclass shouldn't have to think about the subclasses.

Instead, I'll suggest that we add a class attribute like "projection_passthrough=False` to `Blockwise` and then set it to True for all of these classes.  Then future classes can add that attribute and opt-in to this optimization.

Reply:
Good point, added the variable